styles.css

```diff
@@ -92,23 +85,44 @@ img {
   color: #FFF;
   width: 150px;
   height: 65px;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  background-repeat: no-repeat;
```

Comment:
The no repeat was a good find when it comes to displaying the single image, also centering it. Nice work!